### ISSUE TITLE
Zan-Tien Lasso keeps counting

### ISSUE THREAD

**Issue by Hemario** [SacredDuckwhale/Rarity#165]
Even though I have already obtained the mount. I guess this is because I obtained the mount before downloading the new version containing the tracker.

**Comment by SacredDuckwhale** [comment]
Yeah, this is because it's a different item and Rarity doesn't support linking it to the existing mount. It's why I didn't want to add tracking for this, but some people insisted.

I'll look into a way to automatically disable it when the mount was obtained, though you should be able to either mark it as obtained or just disable tracking for it manually. Since I don't have the mount, it might be a little tricky to get this working.
